code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -68,8 +75,8 @@ futureGeneratedArtifacts = enumBulletU $ map foldlSent_
 
 -- | Paragraph describing information encoding
 aboutParagraph5 :: Reference -> Sentence
-aboutParagraph5 infoEncodingWiki = S "As described in the" +:+ namedRef infoEncodingWiki (S "Information Encoding") +:+ S " wiki page, \
-  \Drasil uses specific terminology to address types of \
+aboutParagraph5 infoEncodingWiki = S "As described in the" +:+ namedRef infoEncodingWiki (S "Information Encoding") +:+ S " " +:+ 
```

Comment:
The use of `+:+ S " " +:+` looks like it would add extra whitespace; the `+:+` combinator already adds a space between two `Sentence`s